textacy/spacier/doc_extensions.py

```diff
@@ -481,8 +482,14 @@ def to_bag_of_words(doc, normalize="lemma", weighting="count", as_strings=False)
             Note: The resulting set of frequencies won't (necessarily) sum
             to 1.0, since punctuation and stop words are filtered out after
             counts are normalized.
-        as_strings (bool): if True, words are returned as strings; if False
+        as_strings (bool): If True, words are returned as strings; if False
             (default), words are returned as their unique integer ids
+        remove_stop (bool): If True (default), stop words are removed after
```

Comment:
For the sake of consistency (see: `extract.words()` or `to_terms_list()`), could you rename this arg `filter_stops`? Similarly, `remove_punct` => `filter_punct`.

Reply:
On the same note of consistency: Any objections to adding `filter_nums`?

Reply:
Thanks for the suggestion, I hadn't looked into `extract.words()` in detail.
I've now renamed the new args.